### PR TITLE
Update JS SDK to 0.5.1

### PR DESCRIPTION
**Pull request by fjakobs**

### CODE REVIEW

File: scripts/update_sdk.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+VERSION=$(cat package.json | jq -r '.versions.sdk')
+
+rm -rf vendor 
+mkdir vendor 
+gh release download -R databricks/databricks-sdk-js v${VERSION} -p '*.tgz' 
+mv *.tgz vendor/databricks-sdk-v${VERSION}.tgz 
+
+for F in $(ls packages/*/package.json); do
+    sed -i -e "s/\"@databricks\/databricks-sdk\": \".*\"/\"@databricks\/databricks-sdk\": \"..\/..\/vendor\/databricks-sdk-v${VERSION}.tgz\"/g" $F
+done
```
Comment (kartikgupta-db) [inline scripts/update_sdk.sh:17]: We can rely on yarn to do it potentially. 

```
yarn remove @databricks/databricks-sdk
yarn clean cache && yarn clean cache --mirror
yarn add @databricks/databricks-sdk@file:../../vendo/databricks-sdk-v${VERSION}.tgz
```

**Comment by kartikgupta-db** [comment]
The commit history is messed up. Can you merge from main, to make sure we are not committing something weird.

File: scripts/update_sdk.sh (hunk 1 above)
Comment (fjakobs) [reply scripts/update_sdk.sh:17]: I took your code

**Comment by fjakobs** [comment]
I cleaned the commit history

**Pull request closed without merging**